jira/dc-apps-performance-toolkit-user-guide.md

```diff
@@ -243,10 +242,11 @@ We recommend that you only use this method if you are having problems with the [
     ssh ec2-user@$NODE_IP -o "proxycommand ssh -W %h:%p ec2-user@$BASTION_IP"
     ```
     For more information, go to [Connecting your nodes over SSH](https://confluence.atlassian.com/adminjiraserver/administering-jira-data-center-on-aws-938846969.html#AdministeringJiraDataCenteronAWS-ConnectingtoyournodesoverSSH).
-1. Download the [xml_backup.zip](https://centaurus-datasets.s3.amazonaws.com/jira/8.0.3/large/xml_backup.zip) file.
+1. Download the xml_backup.zip file corresponding to your Jira version.
```

Comment:
Maybe it would be better to leave the link to s3 bucket with different xml_backup.zip files? It might be helpful to end user to quick open it.